modmesh/view.py

```diff
@@ -52,6 +52,8 @@
 try:
     from _modmesh import view as _vimpl  # noqa: F401
     enable = True
+    from PySide6.QtWidgets import QMainWindow
```

Comment:
Please summarize why you need to add this import and variable in comments above this line.

I think it's better to move this action to the pybind11 wrapper of `RWindow`.  You can leave it to me if not sure how and why to do it.

Reply:
I can try to move this action to pybind11 wrapper, but should I finish it in this PR?

Reply:
Let's use another PR for moving it to pybind11.